Miftakhunnurudin

Miftakhunnurudin opened the pull request:
.env

SECRET_KEY = yoursecretkey
API_KEY_WEATHER = bea7899e82e2e085b14050d260ac7a49

Miftakhunnurudin committed:
create Event,user. do some login code

Miftakhunnurudin committed:
CRUD TODO, using env to store secret key, revise login and register

Miftakhunnurudin committed:
add authentication n authorization middleware

Sursev07 commented on server/controllers/eventController.js:
```diff
@@ -0,0 +1,57 @@
+        data.UserId = req.userData.id
+        Event.create(data)
+        .then(result=>{
+            res.status(201).json({messages:'Event Created',result})
```
baca lagi requirement nya yah

Sursev07 commented on server/controllers/eventController.js:
```diff
@@ -0,0 +1,57 @@
+        let id = req.params.id
+        let data = req.body
+
+        Event.update(data,{
```
coba return data nya 
pake returning true

Miftakhunnurudin replied:
kak, aku nggak faham maksudnya bagaimana

Sursev07 commented on server/doc_api.md:
```diff
@@ -0,0 +1,39 @@
+# Todo App Server
```
lengkapin lagi yah

Sursev07 commented on server/models/event.js:
```diff
@@ -0,0 +1,28 @@
+    }
+  };
+  Event.init({
+    title: DataTypes.STRING,
```
validasi jangan lupa yah

Sursev07 commented on server/routes/event.js:
```diff
@@ -0,0 +1,11 @@
+
+router.get('/',EventController.showAllEvent)
+router.post('/',EventController.addEvent)
+router.delete('/:id',authorize,EventController.deleteEvent)
```
bisa dipasang gini juga yah 
```suggestion
router.use(':/id', authorize)
router.delete('/:id',EventController.deleteEvent)
router.put('/:id',EventController.editEvent)
```

Miftakhunnurudin committed:
add eventHandler, revisi

Miftakhunnurudin committed:
add 3rd api weather, dokumentasi masih belum

Miftakhunnurudin committed:
add Client feature:
- form login
- form register
- form todo content
- get weather from server
- get todos from server
- add todo
- delete todo
- show all todos

Miftakhunnurudin committed:
revise :
- documentation api
- event controller =>gettodos => only todo's user can be showed

Miftakhunnurudin committed:
add google auth

Miftakhunnurudin committed:
commit

Miftakhunnurudin committed:
a

Miftakhunnurudin committed:
asad

Miftakhunnurudin committed:
revisi update todo feature in client
